partner_sale_pivot/views/sale_report_views.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="sale_report_view_form" model="ir.ui.view">
+         <field name="name">sale.report.partner_sale_pivot</field>
+         <field name="model">sale.report</field>
+         <field name="arch" type="xml">
+             <pivot disable_linking="True">
+                 <field name="confirmation_date" interval="month" type="col"/>
+                 <field name="product_id" type="row"/>
+                 <field name="price_subtotal" type="measure"/>
```

Comment:
I think it is intereresting to have by default, quantity sold, in a second measure column. Don't you think ?

Reply:
What would be 'quantity sold'?
![Screenshot from 2020-02-04 08-32-28](https://user-images.githubusercontent.com/38267832/73749385-4a0fc480-4729-11ea-892d-c6359076f8ef.png)

Reply:
Oh ! You're right.
I'm so V8.0 ! V12 is more complete. 
Well, I could say Ordered + Delivered + invoiced. but it's maybe to much. Don't no. What do you think ?

Reply:
In my opinion the table become large, also users can get that with a few clicks